src/controller/handler/router.go

```diff
@@ -120,6 +126,42 @@ func wrapper(inner http.HandlerFunc) http.HandlerFunc {
     })
 }
 
+func wrapperWS(inner func (string, int, []byte) ([]byte, error)) http.HandlerFunc {
```

Comment:
Do we really need a wrapper here, we just have only one websocket api.

The below line just for agent send request to server, but our server need to send request to agent ...
m, err = inner(userKey, mT, m)